config/galaxy.ini

```diff
@@ -656,6 +794,12 @@ nglims_config_file = tool-data/nglims.yaml
 # log_events and log_actions functionality will eventually be merged.
 #log_actions = True
 
```

Comment:
This is great, it means that we can use kibana to look at the logs instead of directly. This probably will require some change in the start script and helm chart to insert the kubernetes service name for fluentd when running.